src/tools/data/DataManualExamples/Simulations/updateplots.c

```diff
@@ -215,7 +215,52 @@ void simulate_one_timestep(simulation_data *sim)
         sim->saveCounter++;
 }
 
+/******************************************************************************
+ *
+ * Purpose: This function performs an export to VTK file format.
+ *          Tests changing export options.
+ *
+ * Programmer: Kathleen Biagas 
+ * Date:       September 10, 2021
+ *
+ * Modifications:
+ *
+ *****************************************************************************/
+
+void do_exportVTK(simulation_data *sim)
+{
+    char filename[100];
+    visit_handle vars = VISIT_INVALID_HANDLE;
+    VisIt_NameList_alloc(&vars);
+    VisIt_NameList_addName(vars, "default");
+    /* Add another export variable. */
+    VisIt_NameList_addName(vars, "mesh2d/nodeid");
+
+    {
+        /* Create an option list to change the exported file type */
 
+        visit_handle options = VISIT_INVALID_HANDLE;
+        VisIt_OptionList_alloc(&options);
+        /* FileFormat 0: Legacy ASCII */
+        /* FileFormat 1: Legacy Binary */
+        /* FileFormat 2: XML ASCII */
+        /* FileFormat 3: XML Binary */
+
+        VisIt_OptionList_setValueE(options, "FileFormat", 3);
```

Comment:
This tests *enum* options right. Do we test floats, ints, strings somewhere else? If not, does it make sense to effectively test all those types here?